.jscsrc

```diff
@@ -38,7 +38,7 @@
    "disallowYodaConditions": true,
    "maximumLineLength": 120,
    "requireBlocksOnNewline": true,
-   "requireCamelCaseOrUpperCaseIdentifiers": true,
+   "requireCamelCaseOrUpperCaseIdentifiers": false,
```

Comment:
@AurelioDeRosa Yep, agreed on renaming _request and re-enabling the check for dangling underscores. That's also a good suggestion re: ad hoc overriding where-ever GitHub uses an underscore for a space, will help prevent people from naming things badly. 

For the moment though, can we leave _request as-is and save that for a separate PR? This one is already getting a bit unwieldy. :smile: Will update to re-enable requireCamelCase.